lib/babel-plugins/ti-api.js

```diff
@@ -66,7 +66,8 @@ function plugin(_ref) {
 			return null;
 		}
 
-		const tiNodeRegExp = /^Ti(tanium)?/;
+		// Ensure this is a namespace and not a string starting with Ti. or Titanium.
+		const tiNodeRegExp = /^Ti(tanium)?[.\w+]+$/;
```

Comment:
Could also validate it's not a string litteral?
```JS
if (types.isStringLiteral(member) {
    return null;
}
```
So things like this won't break it
```JS
"Ti.Test".toUpperCase()
```

Reply:
I was able to get this working

```JS
function getTitaniumExpression(member) {

    if (types.isStringLiteral(member.object)) {

        // Prevent picking up strings in evaluation.
        // e.g: "Ti.Test".toUpperCase();
        return null;
    }

    const value = getMemberValue(member);
    if (value === null) {
        return null;
    }

    ...
```

Reply:
Ahhh now I'm following, adding!